### ISSUE TITLE
When babel is loaded, List of Figures is not in all caps

### ISSUE THREAD

**Issue by jcwhittier** [rpspringuel/maine-thesis#23]
Without babel
<img width="400" alt="screen shot 2018-10-19 at 12 16 18 pm" src="https://user-images.githubusercontent.com/2036365/47230717-fee96b00-d398-11e8-9b1a-981faf97ef6a.png">

With babel
<img width="400" alt="screen shot 2018-10-19 at 12 17 14 pm" src="https://user-images.githubusercontent.com/2036365/47230718-ff820180-d398-11e8-8e1c-b935b996272c.png">

**Comment by jcwhittier** [comment]
For babel english users, the following will update the title to LIST OF FIGURES}.
```
\addto\captionsenglish{%
   \renewcommand\listfigurename{LIST OF FIGURES}}


```

**Comment by rpspringuel** [comment]
That would work, but it's probably better to add `\MakeUppercase` in the appropriate place in the class file.  Less for the user to think about.

**Comment by rpspringuel** [comment]
Fixed in ef1030a06b85e51fd97076a39881f896b40f5e13.
